### PR TITLE
Feature/runner k8s

### PR DESCRIPTION
**Pull request by mvxt**
# Description
- Added a page to describe the installation of CircleCI runner on kubernetes
- Added table to easily visualize and read which platforms CircleCI runner is currently supported on

# Reasons
- https://circleci.slack.com/archives/CP4LXVD1T/p1616627698047100

# Screenshots of Changes
image
image
image
image

### CODE REVIEW

File: jekyll/_cci2/runner-on-kubernetes.adoc [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+* Go through the steps to <<runner-installation#authentication,create and authenticate a CircleCI runner resource class>>.
+
+=== Pending Work
+* The Helm chart currently doesn't configure permissions, so the containers will not have elevated privileges. If you need elevated permissions (e.g., for Docker in Docker work, etc.), you will need to modify the chart yourself.
+* The Helm chart currently doesn't support auto-scaling - you will need to modify the `+replicaCount+` parameter inside `+values.yaml+` yourself and re-apply the chart to your cluster.
+* The Helm chart currently expects only one runner resource class type and one token. If you want to configure clusters for multiple runner resource classes, you'll need to set up separate charts for each one.
+
+== Getting Started
+
+. Clone the repository at https://github.com/circleci/circleci-runner-k8s.
```
Comment (stalmy) [inline jekyll/_cci2/runner-on-kubernetes.adoc:37]: Will this location be in CircleCI-Public now?

Comment (mvxt) [reply jekyll/_cci2/runner-on-kubernetes.adoc:37]: Good point, pushing fix for this now.

File: jekyll/_cci2/runner-overview.adoc [hunk 2]
```diff
@@ -34,43 +34,36 @@ There are two key use cases CircleCI aims to meet with the runner:
...
 
-*Preview* CircleCI runners are available on the following platforms:
+=== Supported Platforms Matrix
+
+[.table]
+[cols=8*, options="header"]
+[cols="h,1,1,1,1,1,1,1"]
+|===
+
+| Processor | macOS        | Windows | Ubuntu/Debian | RHEL/CentOS | Other Linux | Docker    | Kubernetes/Other Orchestrator
```
Comment (stalmy) [inline jekyll/_cci2/runner-overview.adoc:62]: This table should match the list that it's replacing

File: jekyll/_cci2/runner-overview.adoc [hunk 3]
```diff
@@ -60,7 +60,7 @@ CircleCI runner is available on multiple platforms. Support levels fall into two
 |===
 
 | Processor | macOS        | Windows | Ubuntu/Debian | RHEL/CentOS | Other Linux | Docker    | Kubernetes/Other Orchestrator
-| x86_64    | Supported    | Preview | Supported     | Supported   | Preview     | Supported | Preview
+| x86_64    | Supported    | Preview | Supported     | Supported   | Preview     | Supported | Supported
```
Comment (stalmy) [inline jekyll/_cci2/runner-overview.adoc:63]: Not sure this is ready to advance past Preview, is it?

**Review by stalmy — APPROVED** [review]

**Comment by mvxt** [comment]
@stalmy See the following updates:
image

image

**Review by stalmy — APPROVED** [review]

**Comment by rosieyohannan** [comment]
@mvxt  I think this feature is out now, can I help get this PR over the line? If it's ready I could read through it later for style etc. and then merge? Let me know :-)

**Comment by mvxt** [comment]
hey @rosieyohannan! I would double check w/ @stalmy on that. I'm no longer with CircleCI so I don't think it's really up to me on whether to merge this or not anymore

**Review by afranco07 — APPROVED** [review]

**Review by tiaandcoffee — APPROVED** [review]
lgtm as well, one non-blocker comment I have is whether a change should be made on the Runner Installation page directing folks to this separate K8s installation page

**Comment by MaT1g3R** [comment]
> lgtm as well, one non-blocker comment I have is whether a change should be made on the Runner Installation page directing folks to this separate K8s installation page

Seems like a good idea, I'll do that

**Review by stalmy — APPROVED** [review]

**Pull request merged**